FMS.Domain/Entities/Facility.cs

```diff
@@ -116,6 +115,8 @@ public Facility(FacilityCreateDto newFacility)
 
         public bool VRPReferral { get; set; } = true;
 
+        public bool HasERecord { get; set; } = true;
```

Comment:
If some of these properties are only used by certain facility types, add some comments to this file, so it's clear to future devs.

Reply:
Done